kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/integration/EmbeddedSchemaRegistry.java

```diff
@@ -0,0 +1,124 @@
+package com.wepay.kafka.connect.bigquery.integration;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.KafkaException;
+import org.apache.kafka.common.serialization.ByteArraySerializer;
+import org.apache.kafka.common.utils.Utils;
+import org.apache.kafka.connect.data.SchemaAndValue;
+import org.apache.kafka.test.TestUtils;
+import org.eclipse.jetty.server.Server;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+import io.confluent.kafka.schemaregistry.rest.SchemaRegistryConfig;
+import io.confluent.kafka.schemaregistry.rest.SchemaRegistryRestApplication;
+import org.apache.kafka.connect.storage.Converter;
+
+import static java.util.Objects.requireNonNull;
+
+public class EmbeddedSchemaRegistry {
```

Comment:
isn't this supposed to be pulled in from the dependencies?

Reply:
Checked with Amit and looks like we need to use RestApp. 
I will update the PR with RestApp. Also, this class actually contains all SR related methods. I will rename it to reflect it is a util.